### ISSUE TITLE
Custom post types not showing up in Category view

### ISSUE THREAD

**Issue by GoogleCodeExporter** [viktorthang/wordpress-custom-content-type-manager#105]
```
What steps will reproduce the problem?
1. Create a custom post type, and enable categories for it.
2. Create a category. Create a post and add it to the category.
3. View the post, click the name of the category under the post.
4. Custom post is not shown in category view.

What is the expected output? What do you see instead?
I expect to see my custom posts on the category page like normal, but instead 
they are nowhere to be found. It either only shows default Posts, or if there 
are no default posts within the category it shows a 'Not Found' page.

Does the problem continue if you disable all other plugins?
Yes.

What version of the plugin are you using?
0.9.0

What version of WordPress?
3.1.4

Please provide any additional information below (e.g. Browser, version,
OS):
Tested with latest Chrome, FF, etc.
Windows 7
```

Original issue reported on code.google.com by `Roryt2...@gmail.com` on 30 Jun 2011 at 10:54

**Comment by GoogleCodeExporter** [comment]
Original comment by `fireproofsocks` on 30 Jun 2011 at 3:21
- Changed state: **InReview**

**Comment by GoogleCodeExporter** [comment]
```
Ok, a prerequisite here is to ensure that URL rewrites are enabled and working 
by creating an .htaccess file (or whatever it is on a Windows server).

Then you create a category for your custom posts, e.g. "mammals", and you 
assign a post to that category.  You expect a URL like the following: 
http://mysite.com/archives/category/mammals  to return a list of posts in the 
"mammals" category, but instead a "Not Found" message is returned (this relies 
on the theme's "category.php" file (by default) -- see 
http://codex.wordpress.org/images/1/18/Template_Hierarchy.png ), which in turn 
uses the loop.php file (via the get_template_part( 'loop', 'category' ); line), 
and the loop.php file is in turn dependent on the have_posts() function, and 
THAT seems to be the root cause of this.

(Deep breath).  So... what to do when WP's have_posts() function sucks?  
Well... what I did was write my own plugin: 
http://code.google.com/p/wordpress-summarize-posts/  

But I need to update it to correct for this... just a sec...
```

Original comment by `fireproofsocks` on 1 Jul 2011 at 5:18
- Changed state: **Verified**

**Comment by GoogleCodeExporter** [comment]
```
Ok, I've just released version 0.6 of my Summarize Posts plugin.  So the 
solution for WP's faulty have_posts() function is this:

1. Download and install the Summarize Posts plugin.
2. Update your theme's category.php file so it looks something like the 
attached category.php

The Summarize Posts plugin will be incorporated into the CCTM plugin at some 
point because it's so damn useful, but for now, it's a separate plugin.   My 
guess is that the limitations of WP's have_posts() function extend into other 
areas (e.g. archives), so it'll become important to supply sample theme files 
that work!
```

Original comment by `fireproofsocks` on 1 Jul 2011 at 3:25
- Changed state: **Fixed**

Attachments:
- [category.php](https://storage.googleapis.com/google-code-attachments/wordpress-custom-content-type-manager/issue-105/comment-3/category.php)

**Comment by GoogleCodeExporter** [comment]
```
 i have the same problem. i created two custom post types and add my post and selected the category while posting things. its showing in wp-admin, but not accessible from the front end, its not showing the category i have chosen when i post it.

help me please, 

http://mywptuts.blogspot.com
```

Original comment by `vjobin...@gmail.com` on 15 Jun 2013 at 4:09
